roles/database/tasks/create_extensions.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Add extensions to DB
+  become_user: postgres
+  postgresql_ext:
+    name: "{{ item }}"
+    db: "{{ database_name }}"
+  with_items:
+    - plpgsql
+    - hstore
+    - pg_trgm
```

Comment:
Now that we're at it I'd add pg_stat_statements. Let me check if we need any other

Reply:
I created issue https://github.com/coopdevs/timeoverflow-provisioning/issues/20 to cover this later, is it OK for you?